Includes/Rosetta/Cards/CardDef.hpp

```diff
@@ -0,0 +1,72 @@
+// This code is based on Sabberstone project.
+// Copyright (c) 2017-2019 SabberStone Team, darkfriend77 & rnilva
+// RosettaStone is hearthstone simulator using C++ with reinforcement learning.
+// Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+#ifndef ROSETTASTONE_CARD_DEF_HPP
+#define ROSETTASTONE_CARD_DEF_HPP
+
+#include <Rosetta/Enchants/Power.hpp>
+
+#include <map>
+#include <vector>
+
+namespace RosettaStone
+{
+//!
+//! \brief CardDef class.
+//!
+//! This class stores the card data such as powers and play requirements.
+//!
+class CardDef
+{
+ public:
+    //! Default constructor.
+    CardDef() = default;
+
+    //! Constructs card def with given \p _power.
+    //! \param _power The power data.
+    CardDef(Power _power);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'CardDef' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/utilForever/RosettaStone/pullRequest?prid=5011091)